digest.gemspec

```
@@ -31,13 +31,34 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = []
   spec.require_paths = ["lib"]
-  spec.extensions    = %w[
-    ext/digest/extconf.rb
-    ext/digest/bubblebabble/extconf.rb
-    ext/digest/md5/extconf.rb
-    ext/digest/rmd160/extconf.rb
-    ext/digest/sha1/extconf.rb
-    ext/digest/sha2/extconf.rb
-  ]
+
+  if RUBY_ENGINE == 'jruby'
```

Use `Gem::Platform` for `--platform` option of `gem build`.
```suggestion
  if Gem::Platform === spec.platform and spec.platform =~ 'java'
```

---

thanks! should we change it here as well? https://github.com/ruby/psych/blob/master/psych.gemspec#L51